locales-en-US.xml

```diff
@@ -159,48 +132,48 @@
 
     <!-- LONG LOCATOR FORMS -->
     <term name="act">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>act</single>
+      <multiple>acts</multiple>						 
     </term>
     <term name="appendix">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>appendix</single>
+      <multiple>appendixes</multiple>						 
     </term>
     <term name="article-locator">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>article</single>
+      <multiple>articles</multiple>						 
     </term>
     <term name="canon">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>canon</single>
+      <multiple>canons</multiple>						 
     </term>
     <term name="elocation">			 
-      <single></single>
+      <single></single> <!-- ??? -->
       <multiple></multiple>						 
     </term>
     <term name="equation">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>equation</single>
+      <multiple>equations</multiple>						 
     </term>
     <term name="rule">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>rule</single>
+      <multiple>rules</multiple>						 
     </term>
     <term name="scene">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>scene</single>
+      <multiple>scenes</multiple>						 
     </term>
     <term name="table">			 
-      <single></single>
-      <multiple></multiple>						 
+      <single>table</single>
+      <multiple>tables</multiple>						 
     </term>
     <term name="timestamp">			 
-      <single></single>
+      <single></single> <!-- ??? -->
```

Comment:
@adam3smith I'm thinking leaving the timestamp terms blank by default? APA style doesn't label timestamps other than by the format (hh:mm:ss). I think that's generally the case with styles?

Reply:
agree.